app/assets/stylesheets/custom.scss

```diff
@@ -79,3 +79,159 @@
   -webkit-margin-end: 0;
   -webkit-margin-start: 0;
 }
+
+
+// profile styling
+
+/* FONTS */
+@import url(http://fonts.googleapis.com/css?family=Open+Sans:400italic,700italic,800italic,400,700,800);
+
+.profile-page {
+  font-family: 'Open Sans', sans-serif;
+  background: #f5f6f7;
+  color: #f5f6f7;
+  font-size: 20px;
+  line-height: 36px;
+  text-align: center;
+  font-weight: 300; }
+
+  .profile-header {
+    background-color: white;
+    padding-top: 60px;
+    padding-left: 50px;
+    padding-bottom: 40px;
+  }
+
+  .profile-page {
+    background-color: #474e5d;
+  }
+
+// .hero-image {
+//   width: auto;
+//   height: auto; }
+//
+.footer-block {
+  height: 180px;
+  margin-top: -90px; }
+
+// .showcase-tiles {
+//   padding-top: 0px;
+//   margin-top: 0px;
+//   display: table-row; }
+//
+// .showcase-info {
+//   display: table-row; }
+//
+// .showcase-info {
+//   text-align: center; }
+//
+// /* =========================
+//
+//     LISTINGS FEED
+//
+//    =========================*/
+
+.listing-slide {
+  margin-top: 20px;
+  margin-bottom: 20px;
+  margin-left: 20px;
+  margin-right: 20px;
+}
+.d-hover {
+  display: none;
+  position: absolute;
+  top: 0;
+  right: 0;
+  left: 0;
+  bottom: 0;
+  text-align: center;
+  padding-top: 25%;
+  line-height: 20px;
+  background: rgba(226, 248, 121, 0.9); }
+  .d-hover p {
+    display: block !important;
+    z-index: 500;
+    margin-top: 25px;
+    font-size: 14px;
+    font-style: italic;
+    font-weight: 300; }
+
+ul.slides {
+  line-height: 0px;
+  padding: 0px; }
+
+ul.slides li {
+display: inline-block;
+width: 25%; }
+
+
+ul.slides li a {
+  display: block;
+  width: 100%;
+  height: 100%;
+  position: relative; }
+
+ul.slides li a:hover .d-hover {
+  display: block; }
+
+ul.slides li a img {
+  width: 100%; }
+
+ul.slides li a p {
+  display: none; }
+
+ul, menu, dir {
+  display: block;
+  list-style-type: disc;
+  -webkit-margin-before: 0px;
+  -webkit-margin-after: 0px;
+  -webkit-margin-start: 0px;
+  -webkit-margin-end: 0px;
+  -webkit-padding-start: 0px;
+}
+
+@media only screen and (max-width: 1000px) {
+      ul.slides li {
+        width: 33%; } }
+    @media only screen and (max-width: 767px) {
+      ul.slides li {
+        width: 50%; } }
+    ul.slides li a {
```

Comment:
Rule declaration should be preceded by an empty line<br>Line should be indented 0 spaces, but was indented 4 spaces<br>Merge rule `ul.slides li a` with rule on line 159<br>Avoid qualifying class selectors with an element.<br>Selector should have depth of applicability no greater than 2, but was 3